src/LedgerLivePlatformSDK/index.ts

```diff
@@ -125,29 +125,44 @@ export default class LedgerLivePlatformSDK {
   /**
    * Start the exchange process by generating a nonce on Ledger device
    * @param {ExchangeType} exchangeType
-   * @param {string} partnerName
    *
    * @returns {Promise<string>} The nonce of the exchange
    */
-  async initExchange(
-    _exchangeType: ExchangeType,
-    _partnerName: string
+  async startExchange(
+    exchangeType: ExchangeType = ExchangeType.FUND
   ): Promise<string> {
-    throw new Error("Function is not implemented yet");
+    return this._request("exchange.start", { exchangeType });
   }
 
   /**
    * Complete an exchange process by passing by the exchange content and its signature.
-   * @param {ExchangePayload} exchangePayload
-   * @param {EcdsaSignature} payloadSignature
-   * @param {FeesLevel} txFeesLevel
+   * We are chaining the success of the triggered device action with a silent sign and broadcast.
+   * @param {ExchangePayload} exchangePayload - Blueprint of the data that we'll allow signing
+   * @param {EcdsaSignature} payloadSignature - Ensures the source of the payload
+   * @param {FeesLevel} feesStrategy - Slow / Medium / Fast
+   * @param {string} provider - Used to verify the signature
+   *
+   * @returns {Promise<SignedTransaction>}
    */
   async completeExchange(
-    _exchangePayload: ExchangePayload,
-    _payloadSignature: EcdsaSignature,
-    _txFeesLevel: FeesLevel
+    provider: string,
+    fromAccountId: string,
+    toAccountId: string,
```

Comment:
From what I understood, this param should be optional because only used for the swap `ExchangeType`.

Speaking of this, how do we make sure that all the parameters are valid and provided?

For example, I could call `startExchange(ExchangeType.SWAP)` and then call `completeExchange` without this optional `toAccountId` param provided (which I should not be able to do) 🤔

Reply:
I need to take a look, perhaps we need the `ExchangeType` on both calls to enforce this, since one has no knowledge of the other 🤔